spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -107,28 +107,24 @@ def restart_lsp(self, language, force=False):
                 try:
                     self.clients_hearbeat[language].stop()
                     client['instance'].disconnect()
+                    client['instance'].stop()
```

Comment:
@goanpeca, here we kill the LSP client if it definitely fails to run.

Reply:
Ok